notebooks/excerpt_overlap_review.ipynb

```diff
@@ -0,0 +1,262 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #24.        )
</pre><p>Could you add some comments to document briefly how this overlap logic works?  It's complicated enough I worry that it will be hard to understand later without comments when we come back to it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Princeton-CDH/ppa-nlp/pull/215/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>